src/components/NetworkOptions/Selector.tsx

```diff
@@ -0,0 +1,116 @@
+import ellipsisLogo from 'assets/svg/ellipsis.svg'
+import Row from 'components/Row'
+import { getChainInfo } from 'constants/chainInfo'
+import { ChevronDown, ChevronUp } from 'react-feather'
+import styled from 'styled-components'
+
+const StyledButton = styled.button<{ isActive: boolean; showChevron?: boolean }>`
+  height: 100%;
+  margin: 0;
+  padding: 6px ${({ showChevron }) => (showChevron ? '4px' : '10px')} 6px 12px;
+  border-radius: 12px;
+  border: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+  background-color: ${({ theme }) => theme.backgroundInteractive};
+  overflow: hidden;
+
+  &::before {
+    content: '';
+    position: absolute;
+    border-radius: inherit;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: ${({ isActive, theme }) => (isActive ? theme.stateOverlayPressed : theme.stateOverlayHover)};
+    pointer-events: none;
+    opacity: ${({ isActive }) => (isActive ? 1 : 0)};
+  }
+
+  &:hover::before {
+    opacity: ${({ isActive }) => (isActive ? 0 : 1)};
+  }
+`
+const SquareChainLogoContainer = styled.div<{ isActive?: boolean }>`
+  border-radius: 8px;
+  width: 24px;
+  height: 24px;
+  outline: 2px solid ${({ theme }) => theme.backgroundInteractive};
+  margin-left: -2px;
+`
+const SquareChainLogo = styled.img`
+  height: 100%;
+  width: 100%;
+  background: #505a78;
+  border-radius: 6px;
+  width: 24px;
+  height: 24px;
+`
+const Chevron = styled.span<{ isActive: boolean }>`
+  padding-top: 1px;
+  color: ${({ theme }) => theme.textSecondary};
+`
+function AllNetworks({ chainIds, isActive }: { chainIds: number[]; isActive: boolean }) {
+  return (
+    <Row data-testid="all-networks-selected">
+      {chainIds.slice(0, 3).map((chainId) => {
+        const chainInfo = getChainInfo(chainId)
+        return (
+          <SquareChainLogoContainer key={chainId} isActive={isActive}>
+            <SquareChainLogo src={chainInfo?.squareLogoUrl} alt={chainInfo?.label} />
+          </SquareChainLogoContainer>
+        )
+      })}
+      <SquareChainLogoContainer isActive={isActive}>
+        <SquareChainLogo src={ellipsisLogo} alt="More network options" />
+      </SquareChainLogoContainer>
+    </Row>
+  )
+}
+export function Selector({
```

Comment:
It's hard to test the styles without having a way to see the component on the web app, can you place it in the pools page behind the `multichain_ux` feature flag?